mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAnnotationMapper.java

```diff
@@ -66,6 +66,7 @@ public void deleteAnnotation(String key) {
     }
 
     public void insertAnnotation(MailboxAnnotation mailboxAnnotation) {
+        Preconditions.checkArgument(!mailboxAnnotation.isNil());
```

Comment:
It's OK, but I have a question: I created AnnotationMapper as the purpose (just to cover the test) but should I keep all struct as it is (Mixing AnnotationMapper on MailboxMapper) or should I separate them all?